### PR TITLE
updates

### PR DESCRIPTION
**Pull request by tcrms**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+   :target: https://pypi.python.org/pypi/anekos
+   :alt: PyPI supported Python versions
+.. image:: https://img.shields.io/pypi/dd/anekos
+   :target: https://pypi.python.org/pypi/anekos
+   :alt: PyPI Downloads
+.. image:: https://img.shields.io/pypi/l/anekos
+   :target: https://pypi.python.org/pypi/anekos
+   :alt: PyPI License
+
+   An unofficial asynchronous wrapper for nekos.life API
```
Comment (NiumXp) [inline README.rst]:
```suggestion
   An asynchronous wrapper for nekos.life API
```

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Features
+========
+
+-  You can download the images! (using
+   `aiofile <https://pypi.org/project/aiofile>`__)
+-  Easy to use with an object-oriented design.
+- Regularly updated with new API updates. (as of 11/06/2022)
```
Comment (NiumXp) [inline README.rst]:
```suggestion
```

File: anekos/__init__.py [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-__version__ = "1.0.0"
-
-from .client import NekosLifeClient
-from .enumeration import NSFWImageTags, SFWImageTags
-from . import result, errors
-from .http_client import Endpoint
+"""
+Async Nekos.Life API Wrapper
+
+An unofficial asynchronous wrapper for nekos.life API
```
Comment (NiumXp) [inline anekos/__init__.py]:
```suggestion
An asynchronous wrapper for nekos.life API
```

File: setup.py [hunk 4]
```diff
@@ -1,12 +1,40 @@
...
+    name='anekos',
+    author='NiumXP',
+    project_urls={
+        'Website': 'https://nekos.life',
+        'Issue tracker': 'https://github.com/Nekos-life/Async-nekos.life-wrapper',
+    },
+    version='1.0.6',
+    packages=['anekos'],
+    license='MIT',
+    description='An unofficial asynchronous wrapper for nekos.life API',
```
Comment (NiumXp) [inline setup.py]:
```suggestion
    description='An asynchronous wrapper for nekos.life API',
```

File: setup.py [hunk 5]
```diff
@@ -1,12 +1,40 @@
...
+    python_requires='>=3.8.0',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'License :: OSI Approved :: MIT License',
+        'Intended Audience :: Developers',
+        'Natural Language :: English',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
```
Comment (NiumXp) [inline setup.py]:
```suggestion
    python_requires='>=3.6.0',
    classifiers=[
        'Development Status :: 5 - Production/Stable',
        'License :: OSI Approved :: MIT License',
        'Intended Audience :: Developers',
        'Natural Language :: English',
        'Operating System :: OS Independent',
        'Programming Language :: Python :: 3.6',
        'Programming Language :: Python :: 3.7',
        'Programming Language :: Python :: 3.8',
        'Programming Language :: Python :: 3.9',
        'Programming Language :: Python :: 3.10',
```

Comment (tcrms) [reply setup.py:35]: does python 3.6.0 support f strings?

Comment (NiumXp) [reply setup.py]: Yes!

`f-string`s has been added in 3.6 according to **[PEP-498](https://peps.python.org/pep-0498/)**

File: anekos/result.py [hunk 6]
```diff
@@ -56,42 +56,19 @@ class ImageResult(Result):
...
-        
-        self.url = url = data.pop("url")
+
+        self.url = url = data.pop('url')
         self.full_name = full_name = basename(url)
         self.name, self.extension = splitext(full_name)
-        self.tag = data.pop("tag")
-        self.bytes = data.pop("bytes", None)
-
-    def sync_save(self, path: str):
```
Comment (NiumXp) [inline anekos/result.py:72]: Why remove `sync_save` and `sync_download`?

File: anekos/__init__.py [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
+"""
+Async Nekos.Life API Wrapper
+
+An asynchronous wrapper for nekos.life API
+"""
+
+__title__ = 'anekos'
+__author__ = 'NiumXP'
+__license__ = 'MIT'
+__version__ = '1.0.1'
```
Comment (NiumXp) [inline anekos/__init__.py]:
```suggestion
__version__ = '2.0.0'
```

File: setup.py [hunk 8]
```diff
@@ -1,13 +1,42 @@
...
+    readme = f.read()
+
+setup(
+    name='anekos',
+    author='NiumXP',
+    project_urls={
+        'Website': 'https://nekos.life',
+        'Issue tracker': 'https://github.com/Nekos-life/Async-nekos.life-wrapper',
+    },
+    version='1.0.6',
```
Comment (NiumXp) [inline setup.py]: Can you add a code to get version from `anekos.__init__.py`?

Something like it:
```python
import re

with open("anekos.__init__.py") as file:
    version = re.search(r"^__version__\s*=\s*[\"']([^\"']*)[\"']", code, re.MULTILINE).group(1)
```

**Review by NiumXp — CHANGES_REQUESTED** [review]
I can't understand why you changed `"` to `'` and added space in default parameters if type-hint is there.

Can you undo the changes in doc-strings about renaming "anekos" to "nekos"?

**Comment by tcrms** [comment]
> I can't understand why you changed `"` to `'` and added space in default parameters if type-hint is there.
> 
> Can you undo the changes in doc-strings about renaming "anekos" to "nekos"?

it was my formatter that i use (blue)

File: anekos/result.py (hunk 6 above)
Comment (tcrms) [reply anekos/result.py:72]: tried both of them and gave error

File: setup.py (hunk 8 above)
Comment (tcrms) [reply setup.py:18]: done

File: anekos/__init__.py [hunk 9]
```diff
@@ -1,15 +1,15 @@
...
-from . import result, errors
-from .http_client import Endpoint
+"""
+Async Nekos.Life API Wrapper
+
+An asynchronous wrapper for nekos.life API
+"""
+
+__title__ = "anekos"
+__author__ = "NiumXP"
```
Comment (NiumXp) [inline anekos/__init__.py]:
```suggestion
__author__ = "NiumXp"
```

File: anekos/http_client.py [hunk 10]
```diff
@@ -13,7 +12,7 @@
 def _make_url(endpoint, parameters):
     url = urljoin(base, endpoint)
     if parameters:
-        url += '?' + '&'.join(f"{k}={quote(v)}" for (k,v) in parameters.items())
+        url += "?" + "&".join(f"{k}={quote(v)}" for (k, v) in parameters.items())
```
Comment (NiumXp) [inline anekos/http_client.py]:
```suggestion
        url += '?' + '&'.join(f"{k}={quote(v)}" for (k, v) in parameters.items())
```

File: anekos/http_client.py [hunk 11]
```diff
@@ -101,18 +101,18 @@ async def endpoints(self):
         _, endpoints_ = await self.get("endpoints")
         for _ in range(len(endpoints_)):
             endpoint = endpoints_.pop(-1)
-            methods, right = endpoint.split('     ')
+            methods, right = endpoint.split("     ")
 
             if right.endswith("-DEPRECATED"):
                 is_deprecated = True
-                url, _ = right.split(' ', 1)
+                url, _ = right.split(" ", 1)
```
Comment (NiumXp) [inline anekos/http_client.py]:
```suggestion
                url, _ = right.split(' ', 1)

```

File: anekos/http_client.py [hunk 12]
```diff
@@ -101,18 +101,18 @@ async def endpoints(self):
...
-                url, _ = right.split(' ', 1)
+                url, _ = right.split(" ", 1)
             else:
                 is_deprecated = False
                 url = right
 
             url = "https://nekos.life" + url
 
-            endpoint = Endpoint(methods.split(','), url, is_deprecated)
+            endpoint = Endpoint(methods.split(","), url, is_deprecated)
```
Comment (NiumXp) [inline anekos/http_client.py]:
```suggestion
            endpoint = Endpoint(methods.split(','), url, is_deprecated)

```

**Review by NiumXp — CHANGES_REQUESTED** [review]

**Comment by NiumXp** [comment]
> > I can't understand why you changed `"` to `'` and added space in default parameters if type-hint is there.
> > Can you undo the changes in doc-strings about renaming "anekos" to "nekos"?
> 
> it was my formatter that i use (blue)

Take care when you commit something that your editor changes, try putting it in a separated commit

**Review by NiumXp — APPROVED** [review]
LGTM!

**Pull request merged**